android/app/build.gradle

```diff
@@ -61,4 +61,5 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.google.firebase:firebase-analytics:17.2.3'
```

Comment:
I am surprised you need this.... it should be added by the plugin automatically. You might try removing it. But it's fine by me if it's required for some reason.

Reply:
https://firebase.google.com/docs/android/setup

Step 4.

Only reason I ask is I have no idea how gradle works.

Reply:
I think you should be able to remove it, those instructions don't apply to Flutter apps.